alpha/veneer/basic/basic.go

```diff
@@ -0,0 +1,80 @@
+package basic
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/alpha/action"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/pkg/image"
+)
+
+type Veneer struct {
+	Registry image.Registry
+}
+
+func (v Veneer) Render(ctx context.Context, ref string) (*declcfg.DeclarativeConfig, error) {
+	// only taking first argument as file
+	stat, serr := os.Stat(ref)
+	if serr != nil {
+		return nil, serr
+	}
+
+	if stat.IsDir() {
+		return nil, errors.New("cannot render veneers by directory reference") // BUGBUG enhance to walk and eat all files in turn, heeding ignore file
```

Comment:
Should this comment stick around?

Reply:
Nope.  Fire-sale time.  
I had thought that Joe would see this comment, go "yes, do that", and away we would go.  
Instead, he said "do you understand the complexity you're enabling by opening up the possibility of collisions .... for an alpha demo project???"  
I took out the rest of the functionality, but left in the comment.